tests/staticfiles_tests/project/documents/cached/source_map.js

```diff
@@ -1 +1,2 @@
 //# sourceMappingURL=source_map.js.map
+let a_variable=1;
```

Comment:
:wink: 
```suggestion
let a_variable = 1;
```

Reply:
It was meant to be minified. 😀 But yeah, OK.